public/js/monitor.js

```diff
@@ -141,25 +162,31 @@ async function hashEmailAndSend(emailFormSubmitEvent) {
   addClass(".input-group-button", "loading-data");
 }
 
-if(document.querySelector(".email-scan")){
-  window.addEventListener("pageshow", function(){
+if (document.querySelector(".email-scan")) {
+  window.addEventListener("pageshow", function() {
     removeClass(".input-group-button", "loading-data");
   });
   document.querySelector(".email-scan").addEventListener("submit", hashEmailAndSend);
   document.querySelector(".input-group-button").addEventListener("click", showMessageIfDisabled);
   const emailToHash = document.querySelector(".email-to-hash");
-  emailToHash.addEventListener("input", enableBtnIfEmailValid);
+  emailToHash.addEventListener("keydown", enableBtnIfEmailValid);
   emailToHash.addEventListener("focusout", enableBtnIfInputEmpty);
-  emailToHash.addEventListener("focusin", function(){
+  emailToHash.addEventListener("focusin", function() {
     removeInvalidMessage();
   });
 }
-window.addEventListener("pageshow", function(){
-  if(document.getElementById("no-breaches") || document.getElementById("found-breaches")){
+
+window.addEventListener("pageshow", function() {
+  if (document.getElementById("no-breaches") || document.getElementById("found-breaches")) {
     handleEvents("Pageview");
   }
 });
-if(document.getElementById("sign-up")){
+
+if (document.getElementById("sign-up")) {
   document.getElementById("sign-up").addEventListener("click", showFalseDoor);
 }
 
+if (document.getElementById("false-door").getAttribute("data-name") === "checkboxChecked") {
```

Comment:
Does every page/template have a `#false-door` element?
If that ever returns `null` or `undefined`, I suspect this would 💥 on the subsequent `.getAttribute("data-name")` bit.

Reply:
I believe every page does have a false door element at this point, but this is a great catch since this may change in the future.